packages/components/src/internal/app/utils.ts

```diff
@@ -235,6 +235,10 @@ export function isAssayRequestsEnabled(moduleContext?: ModuleContext): boolean {
     );
 }
 
+export function isAssayDesignExportEnabled(moduleContext?: ModuleContext): boolean {
+    return hasPremiumModule(moduleContext);
```

Comment:
is this check because we don't actually have anywhere to import an assay design in LKSM hosted only? maybe add a comment

Reply:
Yes, it's meant to be enabled for LKS Starter but not LKSM in general.